rossum_api/models/hook.py

```diff
@@ -1,7 +1,28 @@
 from __future__ import annotations
 
 from dataclasses import dataclass, field
-from typing import Any, Literal
+from typing import Literal
+
+from rossum_api.types import JsonDict
+
+HookAction = Literal[
+    "user_update",  # deprecated in favor of `updated` below
+    "created",
+    "updated",
+    "started",
+    "confirm",
+    "changed",
+    "initialize",
+    "export",
+    "received",
+    "manual",
+    "scheduled",
+    "integration",
+    "interface",
+]
+HookType = Literal["webhook", "function", "job"]
+HookEvent = Literal["annotation_status", "annotation_content", "email", "invocation", "upload"]
+HookExtensionSource = Literal["custom", "rossum_store"]
```

Comment:
Just beware that adding a new value to some enum of options is considered to be a backward-compatible change from our api point of view - https://elis.rossum.ai/api/docs/internal/#versioning